### ISSUE TITLE
In the SideBySide story for OptionMenu, a horizontal scrollbar appears when opening/closing the menu quickly

### ISSUE THREAD

**Issue by openjck** [graph-paper-org/graph-paper#110]

**Comment by openjck** [comment]
Looks like this:

<img width="85" alt="Screen Shot 2020-05-13 at 7 59 23 PM" src="https://user-images.githubusercontent.com/933396/81877880-4c699200-9554-11ea-97fd-bfc012d94b7c.png">

**Comment by hamilton** [comment]
Isn't that actually an errant tooltip?

**Comment by openjck** [comment]
It could be. It's happening on master, so I'll take a look after the OptionMenu PR merges.

**Comment by hamilton** [comment]
I meant that less rhetorically – it is exactly an errant tooltip with no content :)

**Comment by openjck** [comment]
Yeah makes sense. I haven't looked much into it one way or the other since filing the issue, but I'll take your word for it. I don't doubt that it's the tooltip.
